calyx-py/calyx/queue_data_gen.py

```diff
@@ -97,15 +109,37 @@ def dump_json(num_cmds, no_err: bool, queue_size: Optional[int] = None):
         }
     }
 
-    print(json.dumps(commands | values | ans_mem, indent=2))
+    ranks = {
+        "ranks": {
+            "data": [random.randint(0, 400) for _ in range(num_cmds)],
+            # The `values` memory has `num_cmds` items, which are all
+            # random values between 0 and 400.
+            "format": format_gen(32),
+        }
+    }
+
+    times = {
+        "times": {
+            "data": [random.randint(0, 50) for _ in range(num_cmds)],
+            # The `values` memory has `num_cmds` items, which are all
+            # random values between 0 and 400.
+            "format": format_gen(32),
+        }        
+    }
+
+    if nwc:
+        print(json.dumps(commands | values | ranks | times | ans_mem, indent=2))
+    else:
+        print(json.dumps(commands | values | ans_mem, indent=2))
 
 
 if __name__ == "__main__":
     # Accept a flag that we pass to dump_json.
     # This says whether we should use the special no_err helper.
     random.seed(5)
     num_cmds = int(sys.argv[1])
+    nwc = int(sys.argv[2]) == 1
```

Comment:
Instead of making this a 0/1 flag, please make it a string flag like we do for `no-err` and `keepgoing`.